docs/reference/query-language.md

```diff
@@ -11,6 +11,8 @@ The `query` is parsed into a series of terms and operators. There are two types
 
 Multiple terms can be combined together with Boolean operators `AND, OR` to form a more complex query. By default, terms will be combined with the `AND` operator.
 
+IP addresses can be provided as IpV4 or IpV6. It is recommended to use the same format as in the indexed documents.
```

Comment:
Does it change something?

Reply:
There is some unexpected behavior when the format is not the same